seed/utils/geocode.py

```diff
@@ -0,0 +1,118 @@
+# !/usr/bin/env python
+# encoding: utf-8
+
+import requests
+
+from django.conf import settings
+
+from django.contrib.gis.geos import GEOSGeometry
+
+
+def long_lat_wkt(state):
+    if state.long_lat:
+        return GEOSGeometry(state.long_lat,srid=4326).wkt
+
+def geocode_addresses(buildings):
```

Comment:
I abstracted away a lot of the steps into helper methods thinking it would help with readability. I'm not sure if it helped, but feel free to provide feedback on readability (or anything, of course) either way.

Reply:
seems fine to me!